chapters/verilog/operators/reading/README.md

```diff
@@ -88,8 +88,9 @@ Tabelul de mai jos conține operatorii suportați de Verilog, împreună cu nive
 | `+ -` (binari)                      | Adunare, Scădere               | 4          |
 | `<< >> <<< >>>`                     | Shiftare                       | 5          |
 | `< <= > >= == !=`                   | Relaționali                    | 6          |
-| `& ~& ^ ~^ ^~ | ~|`                 | Reducere                       | 7          |
-| `&& ||`                             | Logici                         | 8          |
+| `& ~& ^ ~^ ^~` \| ~\|               | Reducere                       | 7          |
```

Comment:
& = ~& > ^ = ~^ = ^~ > | = ~|

Reply:
Maybe we can use the same row but make a space or a sign between them to show there is a difference.